src/glyphs.js

```diff
@@ -273,4 +351,26 @@ router.post('/:_id/messages/:num', (req, res) => {
   }
 });
 
+// This route is used to import messages from a spreadsheet (CSV)
+router.use(formidable()).post('/import/', (req, res) => {
```

Comment:
Did you test out this line to make sure it did not break any other routes?  My understanding is that calling `use` on a router will proceed to modify the router overall.  So, this would make future routes use this middleware.  Am I wrong?  I'll try to test this out later.

Also, our convention hasn't been to end routes with a trailing `/`.  Is there reason for this nature of the route?